Vars in condition

miamirov opened the pull request.

artem-chupryna reviewed (commented):
Can you provide what is changed in this version, we have removed support for solve_ivp?

miamirov commented:
> Can you provide what is changed in this version, we have removed support for solve_ivp?

Yes,
1) removed solve_ivp option from solver and test cases with it, so now we use only numerous solver
3) add support of all variables instead of states in condition functions
